pkg/cortexpb/cortex.proto

```diff
@@ -21,7 +22,14 @@ message WriteRequest {
   bool skip_label_name_validation = 1000; //set intentionally high to keep WriteRequest compatible with upstream Prometheus
 }
 
-message WriteResponse {}
+message StreamWriteRequest {
+  string TenantID = 1;
+  WriteRequest Request = 2;
+}
+
+message WriteResponse {
+  httpgrpc.HTTPResponse GRPCResponse = 1;
```

Comment:
Is the response gRPC or http? I understand this is just gRPC not http so why we need to wrap it inside `httpgrpc.HTTPResponse`?

It is different from Query Frontend and Querier where we actually need to wrap http response inside a gRPC message

Reply:
Updated WriteResponse to have code and message fields instead